clang/test/dpct/wmma_syclcompat.cu

```diff
@@ -0,0 +1,195 @@
+// clang-format off
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0
+// RUN: dpct --format-range=none -use-syclcompat --use-experimental-features=matrix -out-root %T/wmma_syclcompat %s --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only
+// RUN: FileCheck --input-file %T/wmma_syclcompat/wmma_syclcompat.dp.cpp --match-full-lines %s
+// RUN: %if build_lit %{icpx -c -fsycl -DBUILD_TEST  %T/wmma_syclcompat/wmma_syclcompat.dp.cpp -o %T/wmma_syclcompat/wmma_syclcompat.dp.o %}
+#ifndef BUILD_TEST
+#include <assert.h>
+#include <cuda.h>
+#include <iostream>
+#include <mma.h>
+
+#define WARP_SIZE 32
+
+// MMA matrix tile dimensions.
+
```

Comment:
Please update the commit message on "new added API" to more detail, and add corresponding lit test.